api/src/controllers/articles.ts

```diff
@@ -53,6 +55,16 @@ const articlesController = (
               }
             : undefined,
         },
+        sort:
+          sortBy === "publication.date"
```

Comment:
Should we return a 400 if it's not a value we expect? cf. https://api.wellcomecollection.org/catalogue/v2/works?sort=BLAH

Reply:
Good idea. [Done here](https://github.com/wellcomecollection/content-api/pull/27/commits/7bb1b362c979109e284399410a23d46fee26e521). Might be slightly overkill but wanted it to be readable and flexible.

Reply:
should I also force the `sortOrder` if `sort` is specified or is that alright?